nextcloud_update.sh

```diff
@@ -998,5 +1001,23 @@ Maintenance mode is kept on."
     "Nextcloud update failed!" \
     "Your Nextcloud update failed, please check the logs at $VMLOGS/update.log"
     nextcloud_occ status
+    if [ -n "$SNAPSHOT_EXISTS" ]
+    then
+        # Kill all "$SCRIPTS/update.sh" processes to make sure that no automatic restart happens after exiting this script
+        # shellcheck disable=2009
+        PROCESS_IDS_NEW=$(ps aux | grep "$SCRIPTS/update.sh" | grep -v grep | awk '{print $2}')
+        if [ -n "$PROCESS_IDS_NEW" ]
+        then
+            mapfile -t PROCESS_IDS_NEW <<< "$PROCESS_IDS_NEW"
+            for process in "${PROCESS_IDS_NEW[@]}"
+            do
+                print_text_in_color "$ICyan" "Killing the process with PID $process to prevent a potential automatic restart..."
+                if ! kill "$process"
+                then
+                    print_text_in_color "$IRed" "Couldn't kill the process with PID $process..."
+                fi
+            done
+        fi
```

Comment:
I've copied this from here: https://github.com/nextcloud/vm/blob/caa3b163b8ccadf782d7b055193094b31fb5d2ae/nextcloud_update.sh#L41-L55